src/components/input/DropdownList.tsx

```diff
@@ -39,7 +40,7 @@ const DropdownList = ({
   onChange = () => {},
   placeholder = 'select an option',
   zIndex = 4,
-  width = 'fit-content',
+  width = 150,
```

Comment:
Could this break other dropdowns? Should we keep width fit-content and just set minWidth in addition?

Reply:
Yeah, this was really sad - I had to manually check each of the dropdown since for some reason react-window doesn't support the `min-width` attribute which is such a random shortcoming. I'll see if I can override the CSS manually, but odds are we might have to go with this one